hasura/metadata/actions.yaml

```diff
@@ -216,9 +226,10 @@ custom_types:
   - name: UpdateEpochInput
   - name: Allocation
   - name: Allocations
-  - name: AllocationCsvInput
   - name: CreateUsersInput
   - name: UserObj
+  - name: AllocationCsvInput
```

Comment:
its weird, I have the same issue of this line moving whenever i update the metadata even though i've downgraded the hasura cli,

Reply:
@zashton I have a feeling we merged changes with this line from a user with the CLI upgraded? Not sure how else this is possible